### PR TITLE
Fixed misspelling of variablename

### LINKED ISSUE #78: Search by product id errors out on the manager page

### PR DESCRIPTION
**Pull request by calvinbrown085**
closes #78

### CODE REVIEW

**Comment by ethanphunter** [comment]
:+1:

**Comment by calvinbrown085** [comment]
@ethanphunter fixes this also https://github.com/ethanphunter/PointOfSaleProject/issues/80

File: main.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 app.config.update(dict(
     #DEBUG=True,
-    SECRET_KEY= os.environ["Key"]))
+    SECRET_KEY= """os.environ["Key"]"""))
```
Comment (ethanphunter) [inline main.py]: Key again

Comment (calvinbrown085) [inline main.py]: jesus 1 second

Comment (calvinbrown085) [inline main.py]: fixed @ethanphunter

**Comment by ethanphunter** [comment]
:+1:

**Comment by gilmerc** [comment]
:+1:

**Pull request merged**